src/annotator/guest.coffee

```diff
@@ -115,6 +126,54 @@ module.exports = class Guest extends Delegator
       if not @plugins[name] and @options.pluginClasses[name]
         this.addPlugin(name, opts)
 
+    # Setup event handlers on the root element
+    this._elementEventListeners = []
+    this._setupElementEvents()
+
+
+  # Add DOM event listeners for clicks, taps etc. on the document and
+  # highlights.
+  _setupElementEvents: ->
+    addListener = (event, callback) =>
+      @element[0].addEventListener(event, callback)
+      this._elementEventListeners.push({ event, callback })
+
+    # Hide the sidebar in response to a document click or tap, so it doesn't obscure
+    # the document content.
+    maybeHideSidebar = (event) =>
+      if !this.closeSidebarOnDocumentClick || this.isEventInAnnotator(event) || @selectedTargets?.length
```

Comment:
I never really did grok what `@selectedTargets` were last time I was in this code...